app/src/main/java/org/fossasia/openevent/app/data/speaker/Speaker.java

```diff
@@ -67,4 +68,8 @@
     @Relationship("user")
     @ForeignKey(stubbedRelationship = true, onDelete = ForeignKeyAction.CASCADE)
     public User user;
+
+    @Relationship("session")
+    @ForeignKey(stubbedRelationship = true, onDelete = ForeignKeyAction.CASCADE)
+    public Session session;
```

Comment:
Nope, remove this

Reply:
Yeah, that follows the removal of session